product_docs/docs/ocl_connector/12.1.2.1/01_whats_new.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "What’s New"
+
+legacyRedirectsGenerated:
```

Comment:
these redirect either need to be removed, or changed to reference 12.1.12.1 for each file